config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits.yaml

```diff
@@ -73,25 +59,22 @@ presubmits:
       preset-kind-volume-mounts: "true"
     always_run: true
     optional: true
+    decorate: true
+    extra_refs:
+      - org: kubernetes-sigs
+        repo: cluster-api
+        base_ref: master
+        path_alias: "sigs.k8s.io/cluster-api"
+      - org: kubernetes-sigs
+        repo: image-builder
+        base_ref: master
+        path_alias: "sigs.k8s.io/image-builder"
     spec:
       containers:
-        - image: gcr.io/k8s-testimages/kubekins-e2e:v20191103-6816af1-master
-          args:
-            - "--job=$(JOB_NAME)"
-            - "--root=/go/src"
-            - "--repo=k8s.io/kubernetes=master"
-            - "--repo=sigs.k8s.io/cluster-api=master"
-            - "--repo=sigs.k8s.io/image-builder=master"
-            - "--repo=sigs.k8s.io/cluster-api-provider-gcp=$(PULL_REFS)"
-            - "--repo=sigs.k8s.io/kind=master"
-            - "--service-account=/etc/service-account/service-account.json"
-            - "--upload=gs://kubernetes-jenkins/pr-logs"
-            - "--scenario=execute"
-            - "--"
-            - "bash"
-            - "--"
-            - "-c"
-            - "cd ./../../sigs.k8s.io/cluster-api-provider-gcp && scripts/ci-e2e.sh"
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20191103-6816af1-experimental
```

Comment:
Is experimental actually needed here?

Reply:
picked up the same image as used in `pr-make` as that job seems to be working well.

Reply:
gotcha, I want to say we originally switched to the experimental images when we needed a new version of bazel, so we can likely switch back, but that can be done separately.